.github/workflows/update_iree_requirement_pins.yml

```
@@ -16,45 +21,76 @@ permissions:
   contents: write
   pull-requests: write
 
+env:
+  GIT_BRANCH_NAME: ${{ inputs.branch-name || 'integrates/iree' }}
+
 jobs:
-  update-iree:
+  check-for-existing-branch:
     if: ${{ github.repository_owner == 'nod-ai' || github.event_name != 'schedule' }}
-    runs-on: ubuntu-24.04
+    runs-on: ubuntu-22.04
```

```suggestion
    runs-on: ubuntu-24.04
```
I am aware that all actions in iree-turbine run on `ubuntu-22.04` but I don't see a reason to downgrade here.

---

Thanks, copy-paste strikes again :P